### PR TITLE
Light Account: Block Buy/Sell/Receive

### PR DESCRIPTION
**Pull request by Jon-edge**
image

### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [x] Yes
- [ ] No

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [ ] Tested on iOS device
- [x] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205999316796691

### CODE REVIEW

**Review by swansontec — CHANGES_REQUESTED** [review]
Tiny spacing change is needed. I feel like support will beg us for the call-to-action if we don't add it ourselves, but we can put this if needed.

File: src/components/modals/BackupForTransferModal.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  const { bridge } = props
+
+  const handleClose = () => bridge.resolve()
+
+  return (
+    <ThemedModal bridge={bridge} onCancel={handleClose}>
+      <ModalTitle>{lstrings.backup_title}</ModalTitle>
+      <ModalMessage>{lstrings.backup_for_transfer_message}</ModalMessage>
+    </ThemedModal>
+  )
```
[the remark this answers is not in the document]
Comment (Jon-edge) [reply src/components/modals/BackupForTransferModal.tsx:22]: Major oops

[the remark this answers is not in the document]
Comment (Jon-edge) [reply src/components/modals/BackupForTransferModal.tsx:28]: Added a quick CTA, I'm also betting that we will come back to redesign this to align with the existing backup modal so I'm not doing anything too fancy.

**Comment by Jon-edge** [comment]
I couldn't figure out how to refactor this modal into an action like in `BackupModalActions` since `NavigationBase` doesn't work

**Review by swansontec — APPROVED** [review]

**Pull request merged**